src/macros.rs

```diff
@@ -49,12 +49,13 @@ macro_rules! redis_command {
                 return $crate::raw::Status::Err as c_int;
             }
 
-            if unsafe {
-                $crate::raw::RedisModule_SetCommandACLCategories.unwrap()(
-                    command,
-                    acl_categories.as_ptr(),
-                )
-            } == $crate::raw::Status::Err as c_int
+            if $crate::raw::RedisModule_SetCommandACLCategories.is_some()
```

Comment:
Looks like a candidate of the if let some and else?

Reply:
sure. however that necessitates an `if` within an `if` as the resulting `Some` value will not be usable within the condition of the `if` it was generated from.